src/converter/internal/convertercontroller.cpp

```diff
@@ -101,15 +102,35 @@ mu::Ret ConverterController::fileConvert(const io::path_t& in, const io::path_t&
 
     globalContext()->setCurrentProject(notationProject);
 
+    if (suffix == engraving::MSCZ || suffix == engraving::MSCX) {
```

Comment:
You might want to add `engraving::MSCS` here too (if only for consistency; this extension is not really documented anywhere so early anyone knows about it, but still...)